http/headers/Permissions-Policy.json

```diff
@@ -318,6 +318,41 @@
             }
           }
         },
+        "cross-origin-isolated": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/crossOriginIsolated",
+            "support": {
+              "chrome": {
+                "version_added": "87"
+              },
+              "chrome_android": {
+                "version_added": false
+              },
```

Comment:
per https://chromestatus.com/feature/5690888397258752, support for android is also added